Felienne/hedy
# [BUG FIX] Fixes tab height to max on initialization

**Description**
In this PR we fix an issue where the `max-height` of the adventure panel was only set on switching tabs, not when calling the first one. Resulting in a possible very large view when visiting a level.

**Fixes**
This PR fixes #2292

**How to test**
Navigate to level 2 and verify that there is a max-height and scroll bar, without having to switch tabs.
